### DISCUSSION TITLE
Subject: Urgent: Request for Swift Review of students package Application

### DISCUSSION THREAD

**Discussion by hazemelerefyedu** [community/community#75682] [GitHub Education]
### Hi everyone,

Dear GitHub Support Team,

I trust this message finds you well. I am writing to bring to your attention a matter of urgency regarding a scholarship application that I submitted two days ago. The application was successfully accepted, and it has been under review for an extended period.

Regrettably, I have yet to receive any confirmation via email indicating the status of my application or the approval of the grant. As a university student, this scholarship holds significant importance for me, and I am eager to receive a prompt response.

I kindly request your immediate attention to expedite the review process for my application and provide a swift response regarding the outcome. Your timely assistance in this matter is highly appreciated.

Thank you for your understanding and prompt action.

Sincerely,
[Hazemelerefy]

**Comment by samus-aran** [comment]
Hey @hazemelerefyedu  👋

We've updated our application and re-verification process to ensure that GitHub and our partners can offer exceptional, free benefits to students worldwide during their academic journey.

With this update, when you submit your application, you'll have your academic affiliation verified and gain access to Global Campus right away. Your educational benefits will arrive within 72 hours of this verification. **However, during busy periods like the start of a new semester, it may take a bit longer.**

You’ll get an email notification once your academic benefits are ready, or you can check your status [here](https://education.github.com/discount_requests/pack_application). Meanwhile, as a verified [GitHub Global Campus](https://education.github.com/globalcampus/student) student, you can explore the platform, connect with a local [Campus Expert](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from [GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange) and much more!
